assets/helpers/abTests/helpers/annualContributions.js

```diff
@@ -2,10 +2,130 @@
 
 import { type Store } from 'redux';
 import { contributionSelectionActionsFor } from 'components/contributionSelection/contributionSelectionActions';
+import type { AnnualContributionsTestVariant } from '../abtestDefinitions';
 
-export function setInitialAmountsForAnnualHigherAmountsVariant(store: Store<*, *, *>) {
-  const annualTestVariant = store.getState().common.abParticipations.annualContributionsRoundTwo;
-  if (annualTestVariant === 'annualHigherAmounts') {
-    store.dispatch(contributionSelectionActionsFor('CONTRIBUTE_SECTION').setAmountForContributionType('ANNUAL', 100));
+/* eslint-disable quote-props */
+const numbersInWords = {
+  '25': 'twenty five',
+  '35': 'thirty five',
+  '50': 'fifty',
+  '75': 'seventy five',
+  '100': 'one hundred',
+  '125': 'one hundred and twenty five',
+  '250': 'two hundred and fifty',
+  '300': 'three hundred',
+  '500': 'five hundred',
+  '750': 'seven hundred and fifty',
+};
+/* eslint-enable  quote-props */
+
+const VariantA = {
+  defaults: {
+    GBPCountries: '100',
```

Comment:
the old payment flow logic does not use the `isDefault` field, so this lookup is necessary